gateway/conf.d/apicast.conf

```diff
@@ -115,6 +115,18 @@ location @upstream {
   #   {% endif %}
   # {% endif %}
 
+  # When 'upstream_retry_cases' is empty, apply the same default as NGINX.
+  # If the proxy_next_upstream directive is not declare, the retry policy
```

Comment:
```suggestion
  # If the proxy_next_upstream directive is not declared, the retry policy
```

Reply:
Fixed. Thanks for the review @dfennessy :+1: